movingpandas/trajectory_stop_detector.py

```diff
@@ -16,7 +20,7 @@ class TrajectoryStopDetector:
     at least the specified duration.
     """
 
-    def __init__(self, traj):
+    def __init__(self, traj, n_threads=4):
```

Comment:
wonder if it should default to 1? Someone may run mpd minimally on something like a kubernetes pod with on 1 CPU (2 virtual cores). I think it may be safest where it also follows the current code logic (`self._process_traj_collection(trajs, max_diameter, min_duration`).

Reply:
opened up https://github.com/movingpandas/movingpandas/pull/324 for this

Reply:
Yes, you're probably right.